applications/system/js_app/js_thread.c

```diff
@@ -231,11 +314,32 @@ static int32_t js_thread(void* arg) {
     struct mjs* mjs = mjs_create(worker);
     worker->modules = js_modules_create(mjs, worker->resolver);
     mjs_val_t global = mjs_get_global(mjs);
+    if(worker->path) {
```

Comment:
This is kinda nitpicky, but wouldn't it be better to name these `__dirname` and `__filename` for consistency with Node?

Reply:
It's funny, this was actually the intention when @freehuntx suggested adding this, then @jamisonderek contributed this to momentum with `path` naming and not being js developers we never noticed, and when it was brought up it was too much hassle for what it's worth to change it. Now that we're breaking all scripts with gui and storage and gpio, might aswell change it, true.